Datasets/data-connect-dataset-teamsconversationactivity.md

```diff
@@ -0,0 +1,111 @@
+
+- How many conversations were started by users during a time range? 
+- What is the average number of people leaving or joining a team or a group chat? 
+- How often are the same conversations used for different purposes (i.e., the conversation is renamed)? 
+- How often does a group call happen, and who starts the most calls, or who joins how many calls in a group? 
+
+## Joining with other datasets
+
+The TeamsConversationActivity_v0 dataset can be joined with other activities datasets, to get insights from how employees in the organization interact through outlook and teams. 
+
+## Definitions
+
+- **Signal:**  A Substrate signal is a high-value event that captures a user's behavior. A signal can be either active (like joining a meeting) or passive (like a location change).
+
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| SignalType | string | Name of the signal. | No | None |
+| Id | string | The unique identifier of the Signal. | No | None |
+| StartTime | datetime | The date and time when action happens. | No | None |
+| CreationTime | datetime | The date and time when the signal was created. | Yes | Date |
+| EndTime | datetime | The date and time when action ends. If a signal is a point-in-time action, it is same as StartTime. If the action lasts a time period, this is the action end time captured on client. | No | None|
+| ItemType | string | The type of item being acted on. | No | None |
+| AppName | string | The end user client name being used to generate the signal, E.g. "Unknown", "OutlookMobile", "OWA", "OutlookDesktop", etc. | No | None |
+| AadTenantId | string | AadTenantId of the actor (signal generator). | No | None |
+| ThreadType | string | The type of thread where the conversation is taking place. Among the properties defined by the signal producer. | No |  None |   
+| User_TenantRole | string | Type of user in the tenant (Regular/Admin). Among the properties defined by the signal producer. | No |  None |
+| ThreadName | string | The Channel’s Name. Among the properties defined by the signal producer. | No | None |                    
+| Team_Id | string | Team id, can be chat(for teams chat) or a skype thread id. Among the properties defined by the signal producer. | No |  None |            
+| TeamGroupId | string | Unique GroupId of the teams channel. Among the properties defined by the signal producer. | No | None |                
+| TeamName | string | The team name owning the Channel. Among the properties defined by the signal producer. | No | None |                
+| Emotion | string | Emotion in case the signal is ReactedWithEmoji. Among the properties defined by the signal producer. | No |  None |
+| MembersAdded | object | A list of members added (in case the signal was AddToChannel or Join Channel). Among the properties defined by the signal producer. | No | None |               
```

Comment:
Shouldnt MembersAdded, MembersAdded, MembersAdded  be array type? Since they are Arrays of objects